### DISCUSSION TITLE
Welcome to fpm Discussions!

### DISCUSSION THREAD

**Discussion by milancurcic** [fortran-lang/fpm#279] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  For the maintainers, here are some tips 💡 for getting started with Discussions. We'll leave these in Markdown comments for now, but feel free to take out the comments for all maintainers to see.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by urbanjost** [comment]
Wondering about making an easier way to build a project off the network with a lot of dependencies.  It would be nice if on the network I could do a "fpm build -localize" and the dependencies would be put into a subdirectory like build_local instead of being built; and that I could
then tar up the project and transfer it to a off-network machine; and then build with "fpm build -local" and any reference to an external dependency would look in the build_localize/ directory;  or even better if I have a large collection of local projects like $HOME/fpm_packages/ I could somehow tell fpm to look there first before trying to pull down a package; although git(1) would have to be available to use a specific rev in it's simplest form. Right now the easiest way to convert a project with a lot of dependencies to something self-contained is probably to pull all the dependencies into local subdirectories in the project and change the fpm.toml file to use local directories instead of git repositories;  or install all the packages and then change the references to git local repositories. That is probably not too bad if you have git available  (should work, but have not tried it) but thinking we either need a good description of how to do that (or those); and maybe some feature added to fpm directly or an fpm plugin like fpm-localize that would do that.  If would not just be handy for off-line work; but I think some people when they put out a production version of their code are really going to want some way to create a standalone copy of their code that contains as much as possible of the code so they do not end up not being able to build because a package went away. An official repository would reduce that risk but it is too big a risk at least for now to not capture everything for a production code.  Still have not solidified my thoughts about it; but already several people have asked for help with setting up on off-line machines and do not seem aware of what fpm can do to facilitate that and so do it the hard way like making a new project and copying everything into src, etc. I think it will only get worse as larger projects are created.

**Comment by urbanjost** [comment]
Response files all for abbreviations for long fpm commands, as described in the M_CLI2 package,  are used by adding @NAME to the fpm command line.  Apparently, @ is a special character in powershell. I am not a powershell user but possibly (looking at the online documentation the @ may be escaped with \\ or `; not sure yet.  But wondering if may an alternative like : or # or putting the @ at the end of the name (not sure yet, but @ might only be special at the beginning of a name might work as well; alsp maybe adding a --rsp option as an alternative would be useful.  If anyone knows what would work in powershell (maybe double-quotes or quotes around the name; or has other ideas let me knowl
